web/rootfs/etc/cont-init.d/10-config

```diff
@@ -43,8 +43,11 @@ if [[ $DISABLE_HTTPS -ne 1 ]]; then
             echo "using keys found in /config/keys"
         else
             echo "generating self-signed keys in /config/keys, you can replace these with your own keys if required"
-            SUBJECT="/C=US/ST=TX/L=Austin/O=jitsi.org/OU=Jitsi Server/CN=*"
-            openssl req -new -x509 -days 3650 -nodes -out /config/keys/cert.crt -keyout /config/keys/cert.key -subj "$SUBJECT"
+            openssl req -newkey rsa:2048 -new -x509 -days 9131 -nodes -sha256 \
```

Comment:
do we really need more than 10 years?

Reply:
Why not? The certificate is just to satisfy the browser, not to provide real security.

Reply:
I know, just wondering if the change was deliberate or there was something I was missing.